docs/guides/admin/docs/installation/rpm-el7.md

```diff
@@ -0,0 +1,190 @@
+
+A prepared configuration file for ActiveMQ can be found at `/usr/share/opencast/docs/scripts/activemq/activemq.xml`
+*after Opencast itself has been installed* and should replace `/etc/activemq/activemq.xml`. For an all-in-one
+installation the following command should suffice:
+
+    cp /usr/share/opencast/docs/scripts/activemq/activemq.xml /etc/activemq/activemq.xml
+
+ActiveMQ should be started *prior to* Opencast.
+
+More information about how to properly set up ActiveMQ for Opencast can be found in the [message broker configuration
+documentation](../configuration/message-broker.md).
+
+
+Install Opencast
+------------------
+
+### Basic Installation
+
+For a basic installation (All-In-One) just run:
+
+    yum install opencast<version>-allinone
+
+…where `<version>` is the major version number of the Opencast release you want to install. This will install the
+default distribution of Opencast and all its dependencies.
+
+Now you can start Opencast:
+
+- Don't forget to start configure and start ActiveMQ first as [described in the ActiveMQ installation section
+  ](#install-apache-activemq).
+
+- Then start Opencast by running:
+
+      systemctl start opencast.service
+
+While Opencast is preconfigured, it is strongly recommended to follow at least the [Basic Configuration
+guide](../configuration/basic.md). It will help you to set your hostname, login information, …
+
+
+Advanced Installation
+---------------------
+
+While the basic installation will give you an all-in-one Opencast distribution which is nice for testing, you might
+want to have more control over your system and deploy it over several machines by choosing which parts of Opencast you
+want to install. You can list all Opencast packages with:
+
+    yum search opencast
+
+This will list all available Opencast distributions in the form `opencast<version>-<dist-type>`. Some commonly used
+distributions are:
+
+- `opencast<version>-allinone`
+- `opencast<version>-admin`
+- `opencast<version>-presentation`
+- `opencast<version>-worker`
+
+
+Upgrading Major Versions
+------------------------
+
+Packages will automatically upgrade to the latest minor version in a release series. They do not automatically upgrade
+the latest major version. This is intentional since additional migration steps might be necessary for that. For example,
+if you install `opencast7-admin` you get the latest 7.x release, not the latest 8.x release. To upgrade from one major
+version to another, please consult the upgrade guide for each major version. Still, here is a short overview of the
+required steps:
+
+First, stop Opencast:
+
+    systemctl stop opencast.service
+
+As a reminder, these instructions will upgrade your Opencast installation to a new version which is likely incompatible
+with older versions. If you are performing this on a production system, please ensure you have valid backups prior to
+taking the next steps.
+
+Uninstall your current Opencast package:
+
+    yum remove opencast
```

Comment:
Is this virtual package always going to work regardless of what's actually installed?

Reply:
yes.